crates/fluvio/src/lib.rs

```diff
@@ -207,6 +207,35 @@ pub async fn consumer<S: Into<String>>(
     Ok(consumer)
 }
 
+#[cfg(feature = "otel-metrics")]
```

Comment:
I think there shouldn't any pipeline initialization logic in `fluvio` crate because it is a library. Such methods are end-app level responsibility (user's code or the connector is our case).

Reply:
removed